src/main/java/LiUni/TwitterStatus.java

```diff
@@ -0,0 +1,49 @@
+package LiUni;
+
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+import twitter4j.StatusUpdate;
+import twitter4j.Status;
+
+import java.net.URL;
+
+
+public class TwitterStatus {
+
+    private static final int TWITTER_CHAR_MAX = 280;
+    private static Twitter twitter = TwitterFactory.getSingleton();
+
+    public void postStatus(String text) {
+        if (textErrorCheck(text)) {
+            StatusUpdate newStatus = new StatusUpdate(text);
+            try {
+                Status status = twitter.updateStatus(newStatus);
+                System.out.println("Successfully updated status to [" + status.getText() + "].");
+            }
+            catch (TwitterException e) {
+                System.out.println("Exception occurred. " + e.getErrorMessage());
```

Comment:
you should output the stacktrace

Reply:
Accidentally fixed this on the timeline branch.

Reply:
my current understanding of the branches is 
post tweet branch --> look at post tweet
timeline branch --> look at timeline
please either combine the branches into one or have all of the post tweet information inside this branch.  At a high level, each branch should represent completely finished work that can be merged and tested.

Reply:
I understand; it was an accident that timeline material ended up on post and an accident that I had pushed that change on the wrong branch.